### DISCUSSION TITLE
Q&A can we change oap-server name

### DISCUSSION THREAD

**Discussion by vishalkadam18** [apache/skywalking#11944] [Q&A]
can we change oap-server name in any other name eg. xyz-server??

**Comment by wu-sheng** [comment]
We don't lock the name. How to set the name of your deployment is up to you.

**Reply by vishalkadam18** [reply]
in whole skywalking project where oap-server is there we can change it from everywhere???
we have done this all changes can skywalking agent connect with oap-server or as given name etc...

**Reply by wu-sheng** [reply]
I don't provide that name, my point is, if you want to change, the codes are there for you to change.

**Reply by vishalkadam18** [reply]
can we make seperate build for each jar where every we are building e2e project by this command
`./mvnw clean package -Dmaven.test.skip`

when one module through error then also we have build e2e proejct but is there any way to build seperate module build

**Reply by wu-sheng** [reply]
That is what Maven provided. You should learn from there.

**Reply by vishalkadam18** [reply]
can we build jar or modules???
